types/mertasan__tailwindcss-variables/api.d.ts

```diff
@@ -0,0 +1,17 @@
+import type { PluginOptions } from './internal/PluginOptions';
```

Comment:
```suggestion
import type { PluginOptions } from './index';
```
(if you accept inlining types)